src/main/content/_assets/js/javadocs.js

```diff
@@ -58,6 +58,9 @@ function addExpandAndCollapseToggleButtons() {
         if(isTopLeftPackageIFrame && $(this).contents().find(".toggle").length === 0) {
             var list = $(this).contents().find('ul[title="Packages"]');
             var header = $(this).contents().find("h2[title='Packages']");
+            // header.css("position", "fixed");
+            // header.css("top", "0");
+            // header.css("width", "100%");
```

Comment:
Can you please remove the commented out code?

Reply:
Sure